src/content/faq/General.md

```diff
@@ -8,20 +8,82 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
+
+GraphQL is frequently used for database interactions though. For example, it can be built into a wrapper around a database or a [GraphQL server](https://www.howtographql.com/advanced/1-server/) can query various databases. There are also [services](/code/#services) that implement the GraphQL specification and then allow clients to use GraphQL to query that data source. But it can also be used without a database at all.
+
+For more information on how to get GraphQL to interact with your database, check out our [documentation on resolvers](/learn/execution/#root-fields-resolvers).
+
 ## Does GraphQL replace REST?
 
+No, not necessarily. They both handle APIs and [serve the same purpose](/learn/thinking-in-graphs/#business-logic-layer) from a business perspective. GraphQL is considered an alternative to REST, but it’s not a definitive replacement.
+
+GraphQL and REST can actually co-exist in your stack. For example, you can abstract REST APIs behind a [GraphQL server](https://www.howtographql.com/advanced/1-server/). This can be done by masking your REST endpoint into a GraphQL endpoint using [root resolvers](/learn/execution/#root-fields-resolvers). 
+
+For an opinionated perspective on how GraphQL compares to REST, check out [How To GraphQL](https://www.howtographql.com/basics/1-graphql-is-the-better-rest/).
+
 ## How can I learn GraphQL?
 
+There are many resources available to help you learn GraphQL, including this website. In [our documentation](/learn/), you’ll find a series of articles that explain essential GraphQL concepts and how they work. Our [Community page](/community) is full of resources to reference and groups to join.
+
+For more practical guides, you can visit the [How to GraphQL](https://www.howtographql.com/) fullstack tutorial website. We also have a free online course with edX, [Exploring GraphQL: A Query Language for APIs](https://www.edx.org/course/exploring-graphql-a-query-language-for-apis). 
+
 ## Is GraphQL frontend or backend?
 
+Neither. Instead, it’s [how you can exchange information between the two](https://www.howtographql.com/basics/3-big-picture/). Both the client and the server should understand and communicate through the GraphQL specification.
+
 ## Does GraphQL replace Redux or other state management libraries?
 
+No, GraphQL isn’t a state management library - but it can reduce the need for one.
+
+One benefit of state management libraries like Redux is that they can manipulate API responses into a format that your application understands. With GraphQL, you have control over [what data you request](/learn/queries/#fields) and [how that data is formatted](/learn/execution/). So this benefit is already built-in. Many [client libraries](https://graphql.org/code/#graphql-clients) can also be used to manage state and have features like caching built-in. You may still decide to implement a state management library, but using it to format response data won’t be necessary.
```

Comment:
> and [how that data is formatted](/learn/execution/)
> [...]
> using it to format response data won’t be necessary

We need to be careful not to over-sell this, a lot of GraphQL beginners that I've spoken with get frustrated that they cannot request GraphQL results in custom formats (e.g. flattening trees, forming a key-value map, etc). Sure this could be achieved with things such as [GraphQL-Lodash](https://github.com/APIs-guru/graphql-lodash) but I don't think that's something we can say is intrinsic to GraphQL.

Maybe instead we can go with something softer, such as:

```suggestion
One benefit of state management libraries like Redux is that they can manipulate API responses into a format that your application understands. With GraphQL, you have control over [what data you request](/learn/queries/#fields), and typically results are formatted in a client-friendly way by virtue of the graph design, so this benefit is already built-in. Many [client libraries](https://graphql.org/code/#graphql-clients) can also be used to manage state and have features like caching built-in. You may still decide to implement a state management library, but using it to format response data is generally not necessary.
```

Reply:
So given the previous discussion with @andimarek, I've decided to put this question in the backlog and not include it in this PR. _However_ your notes are great and I've applied those changes to the existing answer so that they are ready whenever we do include it 🙏🏼